src/SPC/builder/unix/library/librdkafka.php

```diff
@@ -25,6 +25,10 @@ function ($lib) {
                 },
                 '--disable-zstd'
             )
+            ->ignoreConfigureArgs(
+                '--with-pic',
+                '--enable-pic',
+            )
```

Comment:
Does this lead to an error? it should only be a warning, which would be fine. There's no point manually specifying ->ignoreConfigureArgs everywhere

Reply:
Ah you're right, it actually errors out instead of printing a warning. How stupid.

Reply:
We should call that ->removeConfigureArgs then, however.